pep-0683.rst

```diff
@@ -0,0 +1,343 @@
+PEP: 683
+Title: Immortal Objects, Using a Fixed Refcount
+Author: Eric Snow <ericsnowcurrently@gmail.com>, Eddie Elizondo <eelizondo@fb.com>
+PEP-Delegate: <TBD>
+Discussions-To: python-dev@python.org
+Status: Draft
+Type: Informational
```

Comment:
There's no API, no (real) user-facing changes, not much controversy, and not a whole lot to decide.  Basically, it is only internal implementation details.  Normally this wouldn't be a PEP but the relevant info is spread around in several email threads, BPO issues, and GH issues.  @encukou asked for an informational PEP to bring the details together for a bit more clarity.

Reply:
Interesting, I'd assume that a change to core refcounting semantics that causes a 4% slowdown would be controversial enough to require SC approval, but I'll trust your judgment.

Reply:
I've added it to the SC agenda.

> @encukou asked for an informational PEP

Hm, it might turn out to need a standards track one. Anyway, IMO the text is most important.

Reply:
Please make it a normal standards track PEP. Thanks!

— Petr, *on behalf of the SC*